studio/src/aem/aem.js

```diff
@@ -7,7 +7,7 @@ const defaultSearchOptions = {
 class AEM {
     #author;
     constructor(bucket, baseUrlOverride) {
-        this.#author = /^author-/.test(bucket);
+        this.#author = Boolean(bucket);
```

Comment:
Because the bucket option didn't seem to follow the pattern `author-`. Instead it's just the environment value. So `this.#author` would be false. 

And that meant in `getFragmentByPath`, `headers` would be empty and the request would fail (actually it just returned a 204).

So I thought I'd change `this.#author` to be true as long as a bucket is provided, but to be honest I'm not completely sure what the purpose of `#author` is and if my change is ok.